airqo_monitor/malfunction_detection/airqo_malfunction_detector.py

```diff
@@ -31,7 +31,8 @@ def get_malfunctions(self, channel_data):
         return malfunction_list
 
     def _sensor_is_reporting_outliers(self, channel_data):
-        """Determine whether the sensor is reporting points outside the reasonable range.
```

Comment:
pep convention says this can be on the same line as the starting comment, but it can also be on the following line, so ok. https://www.python.org/dev/peps/pep-0257/